python/pyarrow/lib.pxd

```diff
@@ -490,7 +490,8 @@ cdef public object pyarrow_wrap_field(const shared_ptr[CField]& field)
 cdef public object pyarrow_wrap_resizable_buffer(
     const shared_ptr[CResizableBuffer]& buf)
 cdef public object pyarrow_wrap_schema(const shared_ptr[CSchema]& type)
-cdef public object pyarrow_wrap_table(const shared_ptr[CTable]& ctable)
+cdef public object pyarrow_wrap_table(
+    object table_class, const shared_ptr[CTable]& ctable)
```

Comment:
I wonder if it's possible to keep the old overload as well (the one-argument form). Does Cython allow this?